### ISSUE TITLE
[KAITO VS Code Experience]

### ISSUE THREAD

**Issue by davidgamero** [Azure/vscode-aks-tools#1070]
**Describe the bug**
when running a test query on successfully deployed phi-3-mini-4k-instruct, i get the following output

```
Error executing curl command: % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0 0 0 0 0 0 0 0 0 --:--:-- 0:00:01 --:--:-- 0 0 0 0 0 0 0 0 0 --:--:-- 0:00:02 --:--:-- 0 0 0 0 0 0 0 0 0 --:--:-- 0:00:02 --:--:-- 0 curl: (7) Failed to connect to localhost port 64063 after 2224 ms: Could not connect to server curl: (3) unmatched brace in URL position 17: generate_kwargs:{temperature:0.5, ^
```

**Mention the platform you are using**
windows 11 vscode

**To Reproduce**
1. Go to Kaito, install Phi-3-mini-4k
2. Click on "test"
3. error appears

**Expected behavior**
get the response from the model

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by tejhan** [comment]
Fixed in PR #1106

**Comment by Tatsinnit** [comment]
❤ Thank you so much for this! under this work: https://github.com/Azure/vscode-aks-tools/pull/1106 this specific issue will be fixed by next release, fyi and cc: @tejhan !! Thanks all, leaving this to open until we release new version sometime next week please.
